src/scripts/main.js

```diff
@@ -1,8 +1,21 @@
 'use strict';
 
 const people = require('./lib/people');
+const table = document.querySelector('.dashboard');
 
-// eslint-disable-next-line no-console
-console.log(people); // you can remove it
+for (const person of people) {
+  const age = (person.died - person.born);
+  const century = Math.ceil(person.died / 100);
+  const sex = person.sex === 'm' ? 'Male' : 'Female';
 
-// write your code here
+  table.children[0].insertAdjacentHTML('beforeend', `
+    <tr>
+      <td>${person.name}</td>
+      <td>${sex}</td>
+      <td>${person.born}</td>
+      <td>${person.died}</td>
+      <td>${age}</td>
+      <td>${century}</td>
+    </tr>
```

Comment:
```suggestion
    <tr>
      <td>${person.name}</td>
      <td>${person.sex === 'm' ? 'Male' : 'Female'}</td>
      <td>${person.born}</td>
      <td>${person.died}</td>
      <td>${person.died - person.born}</td>
      <td>${Math.ceil(person.died / 100)}</td>
    </tr>
```

It's more laconic and you can get rid of redundant variables.